exchanges/exchange_types.go

```diff
@@ -207,3 +207,56 @@ type Base struct {
 	Config        *config.ExchangeConfig
 	settingsMutex sync.RWMutex
 }
+
+// url lookup consts
+const (
+	RestSpot URL = iota
+	RestSpotSupplementary
+	USDTMarginedFutures
+	CoinMarginedFutures
+	Futures
+	Swap
+	Sandbox
+	SpotWsURL
+	SpotWsSupplementaryURL
+	ChainAnalysis
+	EdgeCase1
+	EdgeCase2
+	EdgeCase3
+)
+
+// URL stores uint conversions
+type URL uint32
+
+func (u URL) String() string {
+	switch u {
+	case RestSpot:
+		return "RESTSpotURL"
```

Comment:
Use the same standard as suggested above, we can append URL to the end of these cases.